src/platform/ioc/serviceManager.ts

```diff
@@ -8,7 +8,7 @@ type identifier<T> = string | symbol | Newable<T> | Abstract<T>;
 
 @injectable()
 export class ServiceManager implements IServiceManager {
-    constructor(private container: Container) {}
+    constructor(public readonly container: Container) {}
```

Comment:
Can we create a property with a simple getter that has a decorator to ensure this gets accessed only during tests.
I.e. use the `@testOnlyMethod` decorator that Ian added.

Reply:
Not required, i think its a good practice

Reply:
Good idea. Didn't realize we had a testOnlyDecorator.

Reply:
Yeah, I added it for the WebViewHost I think if you want to see it used. Do wonder about the best was to sometimes use these decorators, like I'm not sure how you would be expected to know that we added on? Util stuff just kinda hide out of the way sometimes.

Reply:
FYI, the decorator doesn't work on a getter. Throws an error. I had to change this to a function ```getContainer```